packages/lit-helpers/test-web/spread.test.js

```diff
@@ -45,6 +50,7 @@ describe('spread', () => {
         number,
         '.array': array,
         '.object': object,
+        '.validity': undefined,
```

Comment:
What's the actual expectation associated with this?

Reply:
With this being set, it should not crash the application, but I can't get the tests to run for some reason. Are you able to for the lit helpers? They didn't seem to be set up at all.